src/pages/build-apps/guides/transaction-signing.md

```diff
@@ -90,6 +90,7 @@ interface STXTransferOptions {
   amount: string;
   memo?: string;
   network: StacksNetwork;
+  fee: number,
```

Comment:
This can be a number or a string.

Reply:
String would be something like "3" for "3 STX" or "3000000" for "3 STX" (in microstacks)?

Reply:
Always µSTX units